src/nbody/nbody.cpp

```diff
@@ -84,7 +87,7 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   Real Rf[3] = {0.0};
   Real Vf[3] = {0.0};
   if (global_frame && (Omf != 0.0) && (qshear != 0.0)) {
-    const Real R0 = std::pow(SQR(Omf) / GM, 1.0 / 3.0);
+    const Real R0 = std::pow(SQR(Omf) / constants.GetGCode(), 1.0 / 3.0);
```

Comment:
Why change this from `GM`? `GM` should already be in code units, so it was fine before right?

Reply:
Yes this was my leftover mistake, thanks for catching this, fixed.